examples/active_fire_detection/README.md

```diff
@@ -0,0 +1,205 @@
+
+Researchers have built and meticulously annotated domain-specific datasets from this vast array of satellite imagery, such as [this dataset](https://drive.google.com/drive/folders/1GIcAev09Ye4hXsSu0Cjo5a6BfL3DpSBm) for space-based active fire detection introduced in [this paper by Pereira et. al](https://arxiv.org/abs/2101.03409).
+
+While these datasets are information-rich and freely available, they are notoriously difficult to collaborate on for machine learning applications due to a few key factors:
+
+##### 1: **Size**
+The SENTINEL-2 and LANDSAT-8 constellations capture imagery of the full global landmass (>57 million mi^2) every 5 and 16 days respectively. That's an enormous amount of data even with regular 3-channel RGB imagery—but these sensors also record outside the visible spectrum, meaning that each image has >10 channels and is as such >3x more storage-intensive than an RGB image of the same pixel dimensions. 
+
+##### 2: Associating data with labels, space, and time
+Satellite images have two extremely crucial pieces of metadata that cannot be ignored during model development: 
+- Where was this image taken? 
+- At what time was this image taken? 
+
+This data is obtained partially through metadata attached to the image files and partially through separate resources issued by the satellite providers, meaning it's often difficult to identify and communicate a single source of truth as a team. 
+
+##### 3: Data quality
+Satellites can't see through clouds! They're also susceptible to having compromised portions due to interference, sun glint, and other unavoidable error sources generated by the curvature and rotation of the earth. 
+
+Taken together, these features should give insight into why model development on satellite data requires active, granular, version-controlled curation of datasets to produce optimal outcomes. 
+
+## Where we're going 
+In this tutorial, we'll transform a widely-cited dataset for wildfires detection from a 10GB Google Drive-hosted .zip file to a cloneable, well-organized [Oxen.ai](www.oxen.ai) data repository ready for model development.
+
+## The Dataset 
+We'll be walking through how to prepare [this excellent dataset](https://drive.google.com/drive/folders/1GIcAev09Ye4hXsSu0Cjo5a6BfL3DpSBm) (introduced in [this paper by Pereira et. al](https://arxiv.org/abs/2101.03409)) for space-based active fire detection.
+
+This dataset contains patches of imagery extracted from LANDSAT-8 satellite imagery across Earth, many of which contain portions of actively burning wildfires. 
+
+While the authors processed the full 1.6TB of imagery captured by LANDSAT-8 in August 2020 (targeting peak fire season), we're primarily interested in the excellent subset of data for which they manually created segmentation masks denoting active burn. 
+
+![An image showing extensive global landmass coverage of the LANDSAT-8 images included in the training set](images/landsat_patches.png)
```

Comment:
Can we put these images side by side? Feels like a lot of scrolling.

Also look at you being so nice to future generative AI data crawls with the alt text 😉